test/unit/composite-view/composite-view.spec.js

```diff
@@ -521,11 +519,6 @@ describe('composite view', function() {
         expect(this.compositeView.$el).to.contain.$text('baz');
         expect(this.compositeView.$el).to.contain.$text('quux');
       });
-
-      it('shound send childView to getChildViewContainer', function() {
```

Comment:
this previously was documented as an internal function despite not being `_` and the API has changed anyhow.. no need to test